radiomesh/gridding.py

```diff
@@ -142,16 +141,20 @@ def impl(
       for bl in range(nbl):
         u, v, w = load_data(uvw, (t, bl), NUVW, -1)
         for ch in range(nchan):
-          # Return early if entire visibility is flagged
+          # Return early if any visibility is flagged
```

Comment:
Why any instead of all here?

Reply:
Matches:

- https://github.com/ratt-ru/radiomesh/blob/87c8d4bc4760fa5f4c43040ff2f7f487ca997656/radiomesh/core.py#L112

and @bennahugo 's policy of treating the entire visibility as suspect if one of the polarisations is flagged